x-pack/docs/en/security/troubleshooting.asciidoc

```diff
@@ -551,6 +551,57 @@ placeholder is used instead. To diagnose the _actual_ problem, you must check
 the {es} logs for further details.
 --
 
+. *Symptoms:*
++
+--
+Authentication in {kib} fails and the following error is printed in the {es}
+logs:
+
+....
+Authentication to realm <saml-realm-name> failed - SAML Attribute [<AttributeName0>] for
+[xpack.security.authc.realms.saml.<saml-realm-name>.attributes.principal] not found in saml attributes
+[<AttributeName1>=<AttributeValue1>, <AttributeName2>=<AttributeValue2>, ...] or NameID [ NameID(format)=value ]
+....
+
+*Resolution:*
+
+This error indicates that {es} failed to find the necessary SAML Attribute in the SAML response that the
+Identity Provider sent. In such a case, {es} is configured as such
```

Comment:
It was not clear to me what "such a case" was referring to. If it's the specific error message example, I think we should say that.  For example:
```suggestion
Identity Provider sent. In this example, {es} is configured as follows:
```